src/combobox/styles/comboBox.m.css

```diff
@@ -4,7 +4,7 @@
 	position: relative;
 	display: inline-block;
 	box-sizing: border-box;
-	width: var(--form-field-width);
+	width: 200px;
```

Comment:
Should we still use some variables for structural styles just so everything looks consistent even when not themed? It would also help maintainability.